docs/SemanticARC.md

```diff
@@ -92,20 +101,28 @@ Once we are able to reason about RC Identity, the next step in implementing Sema
 
    b. If the copy_value instruction has the [take] flag associated with it, then a move is being performed and while a bit by bit copy of the value occurs, no retain_value is applied to it. The original SSA value as a result of this operation has an undefined bit value and in debugging situations could be given a malloc scribbled payload.
 
+[EE] Why would we need a copy_value [take]?
```

Comment:
Yeah, I think we should remove things which are not really needed. Just to keep the SIL as simple as possible.
Adding a flag in an instructions means that possibly many optimizations need to check the flag, which is a source of potential bugs.